content/en/docs/ambient/getting-started/test.sh

```diff
@@ -15,67 +15,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# @setup profile=none
+# @setup profile=ambient
 
 set -e
 set -u
 set -o pipefail
 
-# Kubernetes Gateway API CRDs are required by waypoint proxy.
-snip_download_and_install_2
+source "content/en/docs/ambient/getting-started/deploy-sample-app/snips.sh"
+source "content/en/docs/ambient/getting-started/secure-and-visualize/snips.sh"
+source "content/en/docs/ambient/getting-started/enforce-auth-policies/snips.sh"
+source "content/en/docs/ambient/getting-started/manage-traffic/snips.sh"
+source "content/en/docs/ambient/getting-started/cleanup/snips.sh"
 
-# install istio with ambient profile
```

Comment:
we will have to retain the installation if we go with none profile.. else let me know we can even add an ambient profile CI job for 1.22 if that is what is needed

Reply:
I don't know if we want the two branches to diverge to the point where they have different tests.
If it's possible, I think having the ambient profile CI in 1.22 is the easiest way out